docs/post-mortems/0005-loki-pvc-full.md

```diff
@@ -0,0 +1,133 @@
+
+Example:
+The backend of Oslo Nøkkelen were unavailable for 31 minutes due to an increase in traffic that happened after a
+proaktiv melding.
+-->
+Team Barnehagepris had to postpone batch processing operations for seven hours due to suspended logging.
+
+## Timeline
+<!--
+Describe the relevant activites in a timeline format. Remember activities
+leading up to the event being triggered. Make sure to link to graphs, logs and
+other relevant information sources.
+
+2019-12-30
+    23:30 A proactive message got sendt out to all the citizens of Oslo
+2019-12-31
+    08:23 Oslonøkkelen had been downloaded 30.000 times, distributed evenly across iOS and Android, during the past 3 hours
+    09:09 DOWNTIME START - backend fails due to the increased load
+    09:14 Received Slack notifications due to an increase in 500 status codes in the backend
+    09:16 Initiated an investigation into the issue
+    09:18 EVENT START - Team creates the Slack channel #citykey-incident-backend to better cooperate on the issue
+    09:20 Manually scaling up the number of backend instances
+    09:24 Found the error. The backend fails when it tries to read an item in the database that doesn't exist. Only happens upon a users first login
+    09:28 Pull request with a fix created: http://github.com/oslokommune/something/something/pr/298
+    09:34 Pull request 298 merged and deployed to production
+    09:39 PROBLEM FIXED - Amount of 500 status code requests are decreasing
+    09:40 DOWNTIME END - Amount of 500 status codes are down to zero
+    10:10 EVENT END - Reaches criteria of 30 minutes with normal activity
+-->
+
+| Timeline 20.04.22 | Event                                                                                                                                     |
+|-------------------|-------------------------------------------------------------------------------------------------------------------------------------------|
+| 11:28             | Team Barnehagepris was made aware of the problem                                                                                          |
+| 11:38             | Team Barnehagepris [reached out](https://oslokommune.slack.com/archives/CV9EGL9UG/p1650447510612329) to Team Kjøremiljø about the problem |
+| 11:56             | Team Kjøremiljø initiated support by suggesting different possible solutions                                                              |
+| 19:33             | PROBLEM FIXED - processing resumes.                                                                                                       |
+
+## Impact
+<!--
+Describe the consequences this had for the organization.
+
+Example:
+- Approximately 30.000 users downloaded and experienced an error using the app.
+- Users already logged in experienced some minor delay in the app
+-->
+- Citizens expecting a mail regarding kindergarden benefits experienced an ~eight~ hour delay?
```

Comment:
- During the annual creation of re-applications for citizens with reduced payment the processing had to be halted for ~eight~ hours. 
- Approximately two hours of log data got lost.